src/www/ui/api/Models/UploadSummary.php

```diff
@@ -99,6 +104,7 @@ public function __construct()
     $this->filesCleared = 0;
     $this->clearingStatus = UploadStatus::OPEN;
     $this->copyrightCount = 0;
+    $this->assignee = 1;
```

Comment:
Please change the default value to `null` instead of 1otherwise if one forgets to call `setAssignee()`, 1 will be returned.